setup.py

```diff
@@ -13,7 +13,7 @@
     python_requires=">=3.8",
     install_requires=[
         "prettytable>=0.7.2",
-        "pysha3>=1.0.2",
+        "pycryptodome>=3.15.0",
```

Comment:
Do we need such a strong version requirement? As far as can I see, fairly old versions of the library already support keccak. It might be wise to be flexible with it if it works for what we need, to avoid conflicts with other tools.

Reply:
That is a good idea. What minimum version would you recommend?

Reply:
Going by the changelog I would say `>=3.4.6` (may 2017) as that version [fixed a bug with Keccak](https://pycryptodome.readthedocs.io/en/latest/src/changelog.html#may-2017) and I don't see any newer releases mentioning any changes on Keccak support. Please confirm that everything works with the version we end up choosing, just in case 😄